demos/evp/aesgcm.c

```diff
@@ -101,15 +108,18 @@ void aes_gcm_decrypt(void)
     /* Output decrypted block */
     printf("Plaintext:\n");
     BIO_dump_fp(stdout, outbuf, outlen);
+    /* Compare plaintext against expected */
+    verifyPt = memcmp(gcm_pt, outbuf, sizeof(gcm_pt));
+    printf("Plaintext Verify %s\n", verifyPt == 0 ? "Successful!" : "Failed!");
```

Comment:
Similarly here. This isn't a test program. This doesn't help us demonstrate how to use GCM.